src/ralph/models/edx/server.py

```diff
@@ -63,3 +71,9 @@ def validate_event_type(
         if values.get("event_type") != values.get("context").path:
             raise ValueError("event_type should be equal to context.path")
         return values
+
+
+class BaseServerEventModel(BaseEventModel):
+    """Represents the base server event model all server events inherit from."""
+
+    event_source: Literal["server"]
```

Comment:
Good Idea) Then we don't need to define the `event_source` in the `BaseEventModel` ? Or maybe we could set it to an invalid value like `None` ? And make the `ServerEventModel` inherit the `BaseServerEventModel` ?

Reply:
Yes, I let the server default value at first but as discussed IRL, it would be more logical to overload server value when defining `BaseServerEventModel`

Reply:
To keep homogeneity, I would prefer to remove it from the base model, as we do not define event_type to None in this model. WDYT?

Reply:
Sounds good for me) you could then update the warning message in the doc string of `BaseEventModel`, mentioning that `event_source` is not defined too)

Reply:
You're right, I forgot to pay attention it 🤦